homeassistant/components/alarmdecoder/__init__.py

```diff
@@ -44,8 +41,22 @@
     Platform.SENSOR,
 ]
 
+type AlarmDecoderConfigEntry = ConfigEntry["AlarmDecoderData"]
```

Comment:
Quotes can be removed (you may need to add `from __future__ import annotations` at the top of the file)

```suggestion
type AlarmDecoderConfigEntry = ConfigEntry[AlarmDecoderData]
```

Reply:
It didn't seem to complain without the additional import

Reply:
Resolved in the latest update